src/components/NewMovie/NewMovie.js

```diff
@@ -1,13 +1,100 @@
 import React, { Component } from 'react';
+import './NewMovie.scss';
+import PropTypes from 'prop-types';
 
 export class NewMovie extends Component {
-  state = {};
+  state = {
+    title: '',
+    description: '',
+    imgUrl: '',
+    imdbUrl: '',
+    imdbId: '',
+  };
+
+  submitHandler = (event) => {
+    event.preventDefault();
+
+    this.props.addMovie(this.state);
+
+    this.setState({
+      title: '',
+      description: '',
+      imgUrl: '',
+      imdbUrl: '',
+      imdbId: '',
+    });
+  }
+
+  onChangeHandler = (event) => {
+    const { name, value } = event.target;
+
+    this.setState({
+      [name]: value,
```

Comment:
можно добавить пустую форму просто пробелами 

```suggestion
      [name]: value.trim(),
```